benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -886,16 +895,16 @@ void GraphicsBenchmarkApp::UpdateGUI()
     if (IsXrEnabled()) {
         ImVec2 lastImGuiWindowSize = ImGui::GetWindowSize();
         // For XR, force the diagnostic window to the center with automatic sizing for legibility and since control is limited.
-        ImGui::SetNextWindowPos({(GetUIWidth() - lastImGuiWindowSize.x) / 2, (GetUIHeight() - lastImGuiWindowSize.y) / 2}, 0, {0.0f, 0.0f});
-        ImGui::SetNextWindowSize({0, 0});
+        ImGui::SetNextWindowPos({(GetUIWidth() - lastImGuiWindowSize.x) / 2, (GetUIHeight() - lastImGuiWindowSize.y) / 2}, ImGuiCond_FirstUseEver, {0.0f, 0.0f});
+        ImGui::SetNextWindowSize({512, 512}, ImGuiCond_FirstUseEver);
     }
 #endif
 
     // GUI
     ImGui::Begin("Debug Window");
-    GetKnobManager().DrawAllKnobs(true);
-    ImGui::Separator();
     DrawExtraInfo();
+    ImGui::Separator();
```

Comment:
move the extra info to the first so that there is no need to drag the slider to see those states (on xr devices, we have no control over UI, so have to make this change in order to see the bandwidth states)